homeassistant/generated/integrations.json

```diff
@@ -1044,6 +1044,12 @@
       "config_flow": true,
       "iot_class": "local_polling"
     },
+    "cookidoo": {
+      "name": "Cookidoo",
+      "integration_type": "hub",
```

Comment:
I'm wondering if it's actually a "hub" or rather a "service"? Can you have multiple cookidoos on one account?

Reply:
no, one account per platform/user, --> user=account. Currently it is not even possible on the cookidoo platform to share a "household" without using the same username/password. Pretty old fashioned ;)

Reply:
I had set the type of the manifest to "integration_type": "service" like the `bring` integration. What is the default type? Should I put it explicitely @zweckj ?

Reply:
Default is „hub“. Yes, I think put „service“ in the manifest